### ISSUE TITLE
[DateTimePicker] - Focus and BorderBrush

### ISSUE THREAD

**Issue by marcosgerene** [MahApps/MahApps.Metro#3374]
Hello my friends,

1. How do I do to set focus to a DateTimePicker control (into textbox)?
I tried myControl.Focus(), Keyboard Focus... but its not work!

2. How do I do to change borderbrush to another color?

My code:

![image](https://user-images.githubusercontent.com/15945003/48842599-ce239980-ed7b-11e8-9811-c5bb43282ba9.png)

![image](https://user-images.githubusercontent.com/15945003/48842513-9ae10a80-ed7b-11e8-93ec-34144845b261.png)

![image](https://user-images.githubusercontent.com/15945003/48842621-d7ad0180-ed7b-11e8-928f-5b9d7cdea74a.png)

![image](https://user-images.githubusercontent.com/15945003/48842750-35414e00-ed7c-11e8-8c8d-19fb91348b3b.png)

![image](https://user-images.githubusercontent.com/15945003/48842798-5ace5780-ed7c-11e8-8756-8754630671fd.png)

**Comment by punker76** [comment]
@marcosgerene I'll look to it.

**Comment by marcosgerene** [comment]
@punker76 thanks

**Comment by punker76** [comment]
@marcosgerene 
1) I tried it at the main demo like the following code and it just works as expected
```csharp
        public DateExamples()
        {
            InitializeComponent();

            this.Loaded += DateExamples_Loaded;
        }

        private void DateExamples_Loaded(object sender, System.Windows.RoutedEventArgs e)
        {
            TestDateTimePicker.Focus();
        }
```
2) This was a bug and is now fixed in current source (v2.0 alpha)

Note: I tested this all with the latest source

**Comment by marcosgerene** [comment]
@punker76 great!! I'll update my references. =)

Thank you
